app/views/content/life-as-a-teacher/change-careers/how-to-change-careers-to-become-a-teacher/_article.html.erb

```diff
@@ -0,0 +1,33 @@
+    <p>There are different ways to get QTS.</p>
+    <h3>If you have a degree</h3>
+    <p>If you already have a degree, you can do a postgraduate teacher training course to get QTS.</p>
+    <p>It doesn't matter when you graduated, and your degree subject doesn't necessarily need to be the subject you want to teach. Once you have QTS you could teach various subjects, providing you can show you have enough knowledge.</p>
+    <p>Most postgraduate teacher training courses charge fees. However, there are ways to fund your training, for example, through tuition fee and maintenance loans. You may also be eligible for a bursary or scholarship of up to <%= v :scholarships_generic_maxshortened %>.</p>
+    <p>Other postgraduate teacher training courses come with a salary. This means you will not pay tuition fees and will receive a salary while you train.</p>
+    <p><a href="/train-to-be-a-teacher/if-you-have-a-degree">Find out more about postgraduate teacher training</a>.</p>
+    <h3>If you do not have a degree</h3>
+    <p>If you do not have a degree, you can:</p>
+    <ul>
+      <li>do undergraduate teacher training to get a degree alongside qualified teacher status QTS</li>
+      <li>check if you're eligible to do a teaching degree apprenticeship</li>
+    </ul>
+    <p>Undergraduate teacher training courses can be funded through tuition fee and maintenance loans. You may also be eligible for a bursary of <%= v :bursaries_undergraduate %>.</p>
+    <p>Teaching degree apprenticeships allow you work in a school and earn a salary while getting a bachelor’s degree, so you do not have to pay tuition fees.</p>
+    <p><a href="/train-to-be-a-teacher/if-you-dont-have-a-degree">Find out more about training to teach if you do not have a degree</a>.</p>
+    <h3>If you're unable to get a degree</h3>
+    <p>You do not need a degree and QTS to <a href="/life-as-a-teacher/age-groups-and-specialisms/further-education-teachers">teach in further education</a> or to <a href="/life-as-a-teacher/age-groups-and-specialisms/early-years-teachers">teach in early years</a>. So if you want to become a teacher, but are not able to get a degree, you can explore these options.</p>
+  </section>
+    <div class="col-space-l col-space-l-top">
+    <%= render Content::QuoteWithImageComponent.new(
+      title: "Teacher stories - Claire",
+      text: "<p>As a parent and professional with a regular income, the financial support available to me as a trainee physics teacher made the option of re-training seem not only viable, but attractive.</p><a href='/life-as-a-teacher/change-careers/becoming-a-parent-sparked-my-interest-in-teaching'>Read more about Claire's career change to teaching</a>.",
+      quotes: true,
+      background_color: "blue",
+      heading_color: "pink",
+    ) %>
+  </div>
+</div>
```

Comment:
Worth adding a bit here about events?

Reply:
You read my mind. I was thinking of adding another section to this page about finding out more, and link to the how to become a teacher page, events, and the support page.